src/test/test_bitcoin.cpp

```diff
@@ -74,7 +75,18 @@ BasicTestingSetup::BasicTestingSetup(const std::string& chainName)
     ECC_Start();
     SetupEnvironment();
     SetupNetworking();
-    fPrintToDebugLog = false; // don't want to write to debug.log file
+
+    // Uncomment this to log all errors to stdout so we see them in test output.
+    // We don't enable this by default because several tests intentionally cause
+    // verbose error output (while exercising failure cases).
+    // if (pTracingHandle == nullptr) {
+    //     std::string initialFilter = "error";
+    //     pTracingHandle = tracing_init(
+    //         nullptr, 0,
+    //         initialFilter.c_str(),
+    //         false);
+    // }
```

Comment:
It would be nice to enable this by an environment variable (say `LOG_TO_STDOUT=1`).